docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -150,24 +151,104 @@ On Level 1 systems, the commands in the ``build_<platform>_<compiler>.env`` file
 
 .. code-block::
 
    export <VARIABLE_NAME>=<PATH_TO_MODULE>
    setenv <VARIABLE_NAME> <PATH_TO_MODULE>
 
+.. _MacDetails:
+
+Additional Details for Building on MacOS
+------------------------------------------
+
+.. note::
+    Users not building the SRW App to run on MacOS may skip to the :ref:`next section <BuildExecutables>`. 
+
+The SRW App can be built on MacOS systems, presuming HPC-Stack has already been successfully installed. The following two options have been tested:
+
+* **Option 1:** MacBookAir 2020, M1 chip (arm64, running natively), 4+4 cores, Big Sur 11.6.4, GNU compiler suite v.11.2.0_3 (gcc, gfortran, g++); no MPI pre-installed
+
+* **Option 2:** MacBook Pro 2015, 2.8 GHz Quad-Core Intel Core i7 (x86_64), Catalina OS X 10.15.7, GNU compiler suite v.11.2.0_3 (gcc, gfortran, g++); no MPI pre-installed
+
+.. note::
+    Examples in this subsection presume that the user is running Terminal.app with a bash shell environment. If this is not the case, users will need to adjust the commands to fit their command line application and shell environment. 
+
+The ``build_macos_gnu.env`` script initializes the module environment, lists the location of HPC-Stack modules, loads the meta-modules and modules, and sets compilers, additional flags, and environment variables needed for building the SRW App. The ``$HPC_INSTALL_DIR`` variable is set to the installation directory for the HPC-Stack. The ``srw_common`` file contains a list of specific libraries and modules to be loaded, and it is sourced from ``build_macos_gnu.env``. 
+
+Sample ``build_macos_gnu.env`` contents appear below for Option 1. To use Option 2, the user will need to comment out the lines specific to Option 1 and uncomment the lines specific to Option 2 in the ``build_macos_gnu.env`` file. Additionally, users need to verify that all file paths reflect their system's configuration. 
+
+.. code-block:: console
+
+    # Setup instructions for macOS (build_macos_gnu.env)
+
+    module purge
+    source /opt/homebrew/opt/lmod/init/profile   (Option 1)
+    # source /usr/local/opt/lmod/init/profile    (Option 2)
+    module use $HPC_INSTALL_DIR/modulefiles/stack 
+    module load hpc
+    module load hpc-python
+    module load hpc-gnu
+    module load openmpi
+    module load hpc-openmpi
+
+    export SRW=${HOME}/SRW/ufs-srweather-app
+    module load srw_common
```

Comment:
line 201 "module load srw_common" may need to be commented for the future use, if we decide to use  it for the Tier >1 systems. Instead, we could show the start of the section that loads specific software libraries (but probably no need to show the entire list).  So from line 201 down it will be something like this:
"
# module load srw_common
module load jasper/<version>
module load zlib/<version>
...
module load w3emc/<version>
module load wgrib2/<version>
"

Reply:
So instead of loading srw_common, we would add the list of libraries to load directly into the build_macos_gnu.env file? Should we keep the srw_common section for now and just add a note that users may need to edit the version numbers for software listed in srw_common to reflect their installation of HPC-Stack?